tests/components/unifi/test_controller.py

```diff
@@ -255,28 +262,28 @@ async def test_get_controller_verify_ssl_false(hass):
     with patch("aiounifi.Controller.check_unifi_os", return_value=True), patch(
         "aiounifi.Controller.login", return_value=True
     ):
-        assert await unifi.controller.get_controller(hass, **controller_data)
+        assert await get_controller(hass, **controller_data)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)